cfpb/consumerfinance.gov
# Add dimensions to logo

We don't set an explicit width and height on the logo image in the global header, which Lighthouse flags as possibly contributing to cumulative layout shift (CLS). Even though our CLS scores are really good, we wanted to add `width` and `height` attributes just in case.

---




## Additions

- `width` and `height` attributes to the logo image

## How to test this PR

1. Fire up any page in Chrome and run a Lighthouse report in dev tools
2. Confirm in the report that the logo image does not appear in the "image elements do not have explicit `width` and `height` diagnostic (if you run the report on a page with no other images, as in the screenshots below, that diagnostic shouldn't appear at all)
3. Confirm that the logo still looks OK at all screen sizes

## Screenshots

Lighthouse report on `main` | Lighthouse report on this branch
---- | ----
![lighthouse-before](https://user-images.githubusercontent.com/1862695/135524338-f2351ec0-22ac-4ebe-b6e5-f6eb947654b7.png) | ![lighthouse-after](https://user-images.githubusercontent.com/1862695/135524358-761c07c3-7851-4050-8f5b-9441d628e70e.png)
 
## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
